src/univariate/continuous/ksdist.jl

```diff
@@ -120,16 +120,16 @@ function ceil_rems_mult(n,x)
 end
 
 # n!*(e/n)^n
-function stirling(n)    
+function stirling(n)
     if n < 500
-        s = 1.0
+        s = 1
```

Comment:
Yes. I just put `s = 1.0` back. I had asked Chris to change Float64 literals in the code, but I think he got carried away in some places like this.